ansible/playbooks/roles/ocp-config/README.md

```diff
@@ -0,0 +1,45 @@
+ocp-config: OCP Configuration
+=========
+
+This module will create the ignition files for the cluster nodes. All the ignition files will be hosted at `/var/www/html/ignition/`.
+
+Requirements
+------------
+
+ - The [ocp4-helpernode](https://github.com/RedHatOfficial/ocp4-helpernode) playbook should be executed once on bastion node.
```

Comment:
Why this specific constraint ?

Reply:
Let me rephrase it to "All the required configurations are done on the bastion node eg: HTTP, openshift-install binary is available on $PATH. This can also be achieved by using  [ocp4-helpernode](https://github.com/RedHatOfficial/ocp4-helpernode) playbook."